geoapi/routes/projects.py

```diff
@@ -544,27 +548,9 @@ class ProjectPointCloudResource(Resource):
     @api.marshal_with(point_cloud)
     @project_permissions_allow_public
     def get(self, projectId: int, pointCloudId: int):
-        return PointCloudService.get(pointCloudId)
+        return PointCloudService.get(db_session, pointCloudId)
 
-    @api.doc(id="uploadPointCloud",
```

Comment:
This is the only large out-of-scope change.  I drop this endpoint which allows point clouds to be directly uploaded in a POST.  This endpoint is NOT being used by frontend.  Instead of uploading a file,the current frontends allow users to select to import an existing pointcloud on DesignSafe and then the backend gets the file on their behalf and processes it.

We have some other non-used endpoints where users can upload file directly but none of the frontends use those anymore.  All the frontends use the endpoint where users say which file from DS Tapis we should import for them. 

Really, we should probably drop all the upload routs.

I'm dropping this one here as it made refactoring the use of database sessions by point cloud tasks and methods MUCH easier.